### PR TITLE
Add master node outage chaos scenario

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This adds a new workflow to run master node outage to understand and improve the stability and performance at load.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-cloud-api-4.14-nightly-krkn-hub-cloud-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-cloud-tests-nightly-krkn-hub-cloud-api-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-cloud-tests-nightly-krkn-hub-cloud-api-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-cloud-tests-nightly-krkn-hub-cloud-api-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-cloud-tests-nightly-krkn-hub-cloud-api-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-cloud-tests-nightly-krkn-hub-cloud-api-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-cloud-tests-nightly-krkn-hub-cloud-api-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-cloud-tests-nightly-krkn-hub-cloud-api-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-cloud-tests-nightly-krkn-hub-cloud-api-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-cloud-tests-nightly-krkn-hub-cloud-api-tests

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-cloud-tests-nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: krkn-hub-cloud-api-tests
+  cron: 0 8 */2 * *
+  steps:
+    cluster_profile: aws-chaos
+    env:
+      BASE_DOMAIN: aws.rhperfscale.org
```
[the remark this answers is not in the document]
Comment (chaitanyaenr) [reply ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-cloud-tests-nightly.yaml:35]: So basically this is the domain related to aws-chaos profile in the AWS account to install the OCP cluster.

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-4.14-cloud-tests-nightly-krkn-hub-cloud-api-tests

**Comment by paigerube14** [comment]
This use case/scenario was added in [PR](https://github.com/openshift/release/pull/44680)

**Pull request closed without merging**